schemata/g6-iaas-schema.json

```diff
@@ -0,0 +1,1028 @@
+{
```

Comment:
The Sqlalchemy package uses schemata :)
But yeah I felt like a bit of an arse when I made the folder in the first place and will change it.